packages/cli/cli/src/cli.ts

```diff
@@ -269,18 +274,40 @@ function addGenerateCommand(cli: Argv<GlobalCliOptions>, cliContext: CliContext)
                     description: "Prevent auto-deletion of the Docker containers.",
                 }),
         async (argv) => {
-            await generateWorkspaces({
-                project: await loadProjectAndRegisterWorkspacesWithContext(cliContext, {
-                    commandLineWorkspace: argv.api,
-                    defaultToAllWorkspaces: false,
-                }),
-                cliContext,
-                version: argv.version,
-                groupName: argv.group,
-                shouldLogS3Url: argv.printZipUrl,
-                keepDocker: argv.keepDocker,
-                useLocalDocker: argv.local,
-            });
+            if (argv.api != null && argv.docs != null) {
```

Comment:
this does not handle `fern generate` if neither `--api` nor `--docs` are specified